src/soap/handlers/GetSitesObject.js

```diff
@@ -71,17 +75,15 @@ export async function* getSitesObject(
         organization.data &&
         organization.data.length &&
         organization.data[0]._id
-        ? { organization_id: helpers.orgId(organization.data[0]._id) }
+        ? { organization_id: helpers.org(organization.data[0]._id) }
```

Comment:
Helper call not needed here since `organization.data[0]._id` will be an id and will not be dynamically replaced. The org helper is only to override the query parameter for testing.